tests/selectedtests/conftest.py

```diff
@@ -208,3 +209,8 @@ def _repo(temp_directory):
         return repo
 
     return _repo
+
+
+@pytest.fixture(scope="module")
+def required_builds_regex():
+    return re.compile("!.*")
```

Comment:
It might be a good idea to test with a regex that is different from the parameter default. Otherwise, it could be easy to have a bug where the regex isn't being passed, but using the default.

Reply:
I've added a test now that makes sure when the regex is different that it is passed down correctly all the way